pkg/storage/storage.py

```diff
@@ -226,6 +298,18 @@ def download_model_artifacts(
         self,
         model_name: str,
     ) -> None:
+        """Download a model artifact from an S3 bucket and extracts it.
```

Comment:
We should just delete this method since it's not really going to be used.

Reply:
works for me. Fixed